eng/CredScanSuppression.json

```diff
@@ -31,7 +31,8 @@
         "SecretPlaceholder",
         "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjM2MDB9.adM-ddBZZlQ1WlN3pdPBOF5G4Wh9iZpxNP_fSvpF4cWs",
         "Resource",
-        "sanitized.${expiryReplacement}.sanitized"
+        "sanitized.${expiryReplacement}.sanitized",
+        "123456789"
```

Comment:
Instead of adding a new placeholder can we try to use one of the existing ones?

Reply:
Opps just saw the other conversation around this subject. I would definitely prefer we use an existing placeholder but if there are challenges we can add another.